app/views/instructiveness/assessments/edit.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class='row'>
+  <div class='col-lg-12'>
+    <div class='alert alert-light'>
+      <%= link_to_back(:back) %>
+    </div>
+    <%= render 'assessment_info' %>
+    <div class='card'>
+      <div class='card-header'>
+        <%= fa_icon 'users' %><b><%= t('.students', course_name: @course.name) %></b>
+        <div class='card-header-actions'>
+          <span class='pull-right'>
+            <%= button_tag(t('save'), type: 'submit', class: 'btn btn-outline-success btn-sm', form: 'edit_grades') %>
+          </span>
+        </div>
+      </div>
+      <div class='card-body'>
+        <table class='table table-responsive-sm table-striped table-hover'>
+          <thead class='thead-dark'>
+            <tr>
+              <th style='width: 20%'><%= t('.student_number') %></th>
+              <th style='width: 40%'><%= t('.name') %></th>
+              <th style='width: 15%'><%= t('.year_and_semester') %></th>
+              <th style='width: 15%'><%= t('.group_name') %></th>
+              <th style='width: 10%'><%= @assessment.name %></th>
+            </tr>
+          </thead>
+          <tbody>
+            <%= simple_form_for(@assessment, url: given_course_assessment_path(@course, @assessment), html: { id: :edit_grades }) do |f| %>
+              <%= f.error_notification %>
+              <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
+              <% if f.object.errors.any? %>
+                <ul class='list-group list-group-accent'>
+                  <% f.object.errors.full_messages.each do |message| %>
+                    <li class='list-group-item list-group-item-accent-danger list-group-item-danger'>
+                      <%= message %>
+                    </li>
+                  <% end %>
+                </ul>
+              <% end %>
+              <%= f.simple_fields_for :grades do |grade| %>
+                <% if @enrollments.ids.include?(grade.object.course_enrollment_id) %>
```

Comment:
Buradaki koşulu tam olarak ne amaçla kullandın anlayamadım.

Reply:
@ecmelkytz Anlattığım durum ile ilgili olarak; kayıt (enrollment), yalnızca öğretim görevlisinin yetkisi altında ise not girişi yapmasına izin verilmesi kontrolünü eklemeyi amaçladım. Controller üzerinden kontrol etmeyi denedim ancak sağlayabildiğim en uygun çözüm bu oldu.

Reply:
Bana kalırsa (anlatması bile bir kaç cümle süren) bu tür lojikler view katmanında  olmamalı.